integrated_channels/sap_success_factors/utils.py

```diff
@@ -275,6 +276,21 @@ def resolve_removed_courses(self, previous_audit_summary):
     }
 
 
+def get_enterprise_course_enrollment_url(enterprise_customer, course_id):
+    """
+    Given an EnterpriseCustomer and a course ID, craft a URL that links to the enterprise enrollment landing
+     page for that course.
+
+    Args:
+        enterprise_customer (EnterpriseCustomer): The EnterpriseCustomer that a URL needs to be built for
+        course_id (str): The string identifier of the course in question
+    """
+    try:
+        return enterprise_customer.get_course_enrollment_url(course_id)
+    except NotConnectedToOpenEdX:
```

Comment:
The fact that we have to catch this for a configuration check is incredibly frustrating to me. I've filed [ENT-428](https://openedx.atlassian.net/browse/ENT-428) to handle this and make for a happier overall path.